SBOM/spdx.md

```diff
@@ -0,0 +1,216 @@
+                },
+                {
+                    "annotationDate": "2023-06-19T10:49:04.840659Z",
+                    "annotationType": "OTHER",
+                    "annotator": "Tool: alma-sbom 0.0.1",
+                    "comment": "almalinux:albs:build:ID=5636"
+                },
+                {
+                    "annotationDate": "2023-06-19T10:49:04.840687Z",
+                    "annotationType": "OTHER",
+                    "annotator": "Tool: alma-sbom 0.0.1",
+                    "comment": "almalinux:albs:build:URL=https://build.almalinux.org/build/5636"
+                },
+                {
+                    "annotationDate": "2023-06-19T10:49:04.840715Z",
+                    "annotationType": "OTHER",
+                    "annotator": "Tool: alma-sbom 0.0.1",
+                    "comment": "almalinux:albs:build:author=eabdullin1 <55892454+eabdullin1@users.noreply.github.com>"
+                },
+                {
+                    "annotationDate": "2023-06-19T10:49:04.840743Z",
+                    "annotationType": "OTHER",
+                    "annotator": "Tool: alma-sbom 0.0.1",
+                    "comment": "almalinux:albs:build:source:gitURL=https://git.almalinux.org/rpms/bash.git"
+                },
+                {
+                    "annotationDate": "2023-06-19T10:49:04.840770Z",
+                    "annotationType": "OTHER",
+                    "annotator": "Tool: alma-sbom 0.0.1",
+                    "comment": "almalinux:albs:build:source:type=git"
+                },
+                {
+                    "annotationDate": "2023-06-19T10:49:04.840798Z",
+                    "annotationType": "OTHER",
+                    "annotator": "Tool: alma-sbom 0.0.1",
+                    "comment": "almalinux:albs:build:source:gitCommit=1a3ebc6398c31698863ba9b31a9f5effdb425c4f"
+                },
+                {
+                    "annotationDate": "2023-06-19T10:49:04.840826Z",
+                    "annotationType": "OTHER",
+                    "annotator": "Tool: alma-sbom 0.0.1",
+                    "comment": "almalinux:albs:build:source:gitRef=imports/c9/bash-5.1.8-6.el9_1"
+                },
+                {
+                    "annotationDate": "2023-06-19T10:49:04.840865Z",
+                    "annotationType": "OTHER",
+                    "annotator": "Tool: alma-sbom 0.0.1",
+                    "comment": "almalinux:albs:build:source:gitCommitCasHash=70c5a02df7074eada535aa35643f019574f7005ff345fd6f482eca797ae7c1d8"
+                }
+            ],
+            "builtDate": "2023-01-24T09:44:30.238513+00:00Z",
+            "checksums": [
+                {
+                    "algorithm": "SHA256",
+                    "checksumValue": "1bfcf5bf3187ff21db8109d48a0198b92341a9124f63db82cf501d1968ba1374"
+                }
+            ],
+            "downloadLocation": "pkg:rpm/almalinux/bash@5.1.8-6.el9_1?arch=x86_64&upstream=bash-5.1.8-6.el9_1.src.rpm",
```

Comment:
We need to figure out whether this is mandatory, as I see:

If the reference is BINARY, its relationship to `PackageDownloadLocation` might need to be explained. If the reference is SOURCE, its relationship to `PackageDownloadLocation` and `SourceInformation` might need to be explained.
We're already providing a purl, isn't that already enough?

If need to include it:
 * shouldn't this be `packageDownloadLocation`?
 * do we need to come up with a solution for providing the right `downloadLocation`? what we have when generating the sbom is the artifact that came off the ALBS. Or, are we okay with just providing the info from that one?

Reply:
Okay, that makes sense, let's just set it to `NONE` for now.
And yes, got confused about `downloadLocation` vs `packageDownloadLocation` when reading the spec back and forth :sweat_smile:

Reply:
I just added a commit that sets the `downloadLocation` to `NONE` until we have something better.